distribution/kernel/carbon-home/repository/resources/conf/default.json

```diff
@@ -224,5 +224,6 @@
   "jce_provider.provider_name" : "BC",
   "signature_util.enable_sha256_algo" : true,
   "tenant_mgt.enable_input_validation" : false,
-  "remote_logging.config_sync.period": "15"
+  "remote_logging.config_sync.period": "15",
+  "legacy_authz_runtime.enable": false
```

Comment:
In the initial PR set this to true
and send a separate PR to set enable = false once all features are ready

Reply:
fixed